src/services/addressResolver/index.ts

```diff
@@ -42,13 +46,42 @@ export const getIdentityName = async (
 ): Promise<ResolutionResult | undefined> => {
   const identityName = await addresses.reduce(async (accumulator, currentValue) => {
     if (await accumulator) return accumulator;
-    const result = await resolveAddressNameByEndpoint(
-      currentValue.endpoint + address,
-      currentValue.apiHeader,
-      currentValue.apiKey
-    );
+    if (!currentValue.path.addressResolution) return undefined;
+    const url = getPath(join(currentValue.path.addressResolution, address), currentValue.endpoint);
+    const result = await resolveAddressNameByEndpoint(url, currentValue.apiHeader, currentValue.apiKey);
+    if (!result) return undefined;
     return { result, source: currentValue.name };
   }, Promise.resolve<ResolutionResult | undefined>(undefined));
 
   return identityName;
 };
+
+interface FeatureResponse {
+  features: {
+    addressResolution?: {
+      location: string;
+    };
+    entityLookup?: {
+      location: string;
+    };
+  };
+  version: number;
+}
+
+export const getFeatures = async (url: string, apiHeader?: string, apiKey?: string): Promise<FeatureResponse> => {
+  try {
```

Comment:
this http call logic seems to be repeated across resolveAddressNameByEndpoint and getFeatures, i think maybe we should extract it?